serverless/serverless-kafka.adoc

```diff
@@ -36,6 +36,14 @@ spec:
 // Install Kafka
 include::modules/serverless-install-kafka-odc.adoc[leveloffset=+1]
 
+// Configure TLS and SASL for Kafka
+== Configuring authentication for Kafka
+
+In production, Kafka clusters are often secured using the TLS or SASL authentication methods. This section shows how to configure the Kafka channel to work against a protected Red Hat AMQ Streams (Kafka) cluster using TLS or SASL.
+
+include::modules/serverless-kafka-tls.adoc[leveloffset=+1]
+include::modules/serverless-kafka-sasl.adoc[leveloffset=+1]
```

Comment:
To adjust the module heading level, you can change these include statements to `[leveloffset=+2]`. Also, module titles should always be `= Title of module` vs `== Title of module`.